docs/src/views/ColorPalette.tsx

```diff
@@ -25,20 +59,39 @@ const ColorPalette = () => (
     }}
   >
     <ProviderConsumer
-      render={({ siteVariables: { colors, contextualColors, emphasisColors, naturalColors } }) => (
-        <DocPage title="Color Palette">
+      render={({ siteVariables: { colors, emphasisColors, naturalColors } }) => (
+        <DocPage title="Colors">
           <Header as="h2">Introduction</Header>
           <p>
-            The color palette for a theme has many requirements and constraints. There is a need to
-            be intentional and functional with color use. We analyzed existing frameworks and picked
-            the best ideas from them.
+            The organizing of the colors for an application has many requirements and constraints.
+            There is a need to be intentional and functional with color use. We analyzed existing
+            frameworks and picked the best ideas from them. In Stardust, the colors mechanisms are
+            completely based on the <code>siteVariables</code>, we don't have any additional API
+            specific to the colors.
           </p>
           <p>
-            Each theme should match our color palette types fully. This will allow you to use our
-            theming features completely and keep your palette structured.
+            There are two things important in order for colors to work transparently when the theme
+            switching is in play:
           </p>
+          <ul>
+            <li>
+              <b>{link('Color scheme', '#color-scheme')}</b> - design tokens for all colors used in
```

Comment:
I would swap **Color scheme** with **Color palette** so that the order corresponds with the explanation below.